code/modules/vehicles/scooter.dm

```diff
@@ -0,0 +1,87 @@
+/obj/vehicle/scooter
+	name = "scooter"
+	desc = "A popular child's toy back on the planets, handcrafted by some greyshirt."
```

Comment:
Oh, come on. Greyshirts are not the only people who will want these - Engineers, who have the require materials at all times, would also make these. I assume the same for cargo.